androidshared/src/main/java/org/odk/collect/androidshared/ui/SnackbarUtils.kt

```diff
@@ -103,7 +107,13 @@ object SnackbarUtils {
                     dismiss()
                 }
             }
-        }.show()
+        }.addCallback(object : BaseTransientBottomBar.BaseCallback<Snackbar>() {
+            override fun onDismissed(transientBottomBar: Snackbar?, event: Int) {
+                super.onDismissed(transientBottomBar, event)
```

Comment:
You might have already done this, but I think there's a couple of situations that we should double check that this gets called in:
1. When the Snackbar is dismissed after being on screen for its duration
2. When the Snackbar is dismissed manually
3. When the Activity is destroyed before the Snackbar is dismissed

Reply:
I've checked those cases and it seems to work well.